src/Symfony/Component/Validator/Tests/Constraints/ExpressionValidatorTest.php

```diff
@@ -235,4 +235,17 @@ public function testExpressionLanguageUsage()
 
         $this->assertTrue($used, 'Failed asserting that custom ExpressionLanguage instance is used.');
     }
+
+    public function testExpressionLanguageUsageWithCustomDataPath()
```

Comment:
I orientated me on the naming from the test function above with is name `testExpressionLanguageUsage `. But I can change it for better unterstanding